src/Identifier/IdentifierInterface.php

```diff
@@ -18,10 +18,10 @@
     /**
      * Identifies an user or service by the passed credentials
      *
-     * @param mixed $credentials Authentication credentials
-     * @return \ArrayAccess|null
+     * @param array $credentials Authentication credentials
+     * @return mixed
```

Comment:
I did `git pull --rebase origin master`. `git rebase origin/master` is telling me `Current branch fix-password-identifier is up to date.`